### ISSUE TITLE
dump_indexeddb_details ModuleNotFoundError

### ISSUE THREAD

**Issue by kumavis** [obsidianforensics/hindsight#115]
trying to take this for a spin `dump_indexeddb_details.py`
```
➜ python3 ./pyhindsight/lib/ccl_chrome_indexeddb/dump_indexeddb_details.py 
Traceback (most recent call last):
  File "/home/xyz/Development/hindsight/./pyhindsight/lib/ccl_chrome_indexeddb/dump_indexeddb_details.py", line 3, in <module>
    import ccl_chromium_indexeddb
  File "/home/xyz/Development/hindsight/pyhindsight/lib/ccl_chrome_indexeddb/ccl_chromium_indexeddb.py", line 34, in <module>
    import ccl_leveldb
  File "/home/xyz/Development/hindsight/pyhindsight/lib/ccl_chrome_indexeddb/ccl_leveldb.py", line 34, in <module>
    from pyhindsight.lib.ccl_chrome_indexeddb import ccl_simplesnappy
ModuleNotFoundError: No module named 'pyhindsight.lib'
```

im not a python dev so i dont really know what im doing but seems bad import is [here](https://github.com/obsidianforensics/hindsight/blob/6b754e216b9cbc522ea4c6e4614012ab5fd75872/pyhindsight/lib/ccl_chrome_indexeddb/ccl_leveldb.py#L34)

**Comment by jJit0** [comment]
I figured it out by commenting this line
`#pyhindsight.lib.ccl_chrome_indexeddb import ccl_simplesnappy`

**Comment by obsidianforensics** [comment]
Thanks for flagging this - after looking closer, I had not included that "lib" directory in the PyPI package, so that's one issue I've fixed. It looks like yours is something different, I'll have to take closer look at the way the imports work.

**Comment by Pysis868** [comment]
```
⋊> python $HOME/.local/bin/hindsight.py
Traceback (most recent call last):
  File "$HOME/.local/bin/hindsight.py", line 21, in <module>
    from pyhindsight.analysis import AnalysisSession
  File "$HOME/.local/lib/python3.11/site-packages/pyhindsight/analysis.py", line 12, in <module>
    from pyhindsight.browsers.chrome import Chrome
  File "$HOME/.local/lib/python3.11/site-packages/pyhindsight/browsers/chrome.py", line 17, in <module>
    import pyhindsight.lib.ccl_chrome_indexeddb.ccl_blink_value_deserializer
ModuleNotFoundError: No module named 'pyhindsight.lib'

⋊> python $HOME/.local/bin/hindsight_gui.py
Traceback (most recent call last):
  File "$HOME/.local/bin/hindsight_gui.py", line 10, in <module>
    from pyhindsight.analysis import AnalysisSession
  File "$HOME/.local/lib/python3.11/site-packages/pyhindsight/analysis.py", line 12, in <module>
    from pyhindsight.browsers.chrome import Chrome
  File "$HOME/.local/lib/python3.11/site-packages/pyhindsight/browsers/chrome.py", line 17, in <module>
    import pyhindsight.lib.ccl_chrome_indexeddb.ccl_blink_value_deserializer
ModuleNotFoundError: No module named 'pyhindsight.lib'

⋊> python --version
Python 3.11.2
⋊> pip --version
pip 22.2.2 from /usr/lib/python3.11/site-packages/pip (python 3.11)
```
I just want a leveldb viewer that works today.
Tried:
- https://github.com/markmckinnon/Leveldb-py
- https://github.com/mohamedkomalo/leveldb-gui-client
- https://softwarerecs.stackexchange.com/questions/14409/leveldb-database-client
Now this, maybe more.

Other info and issues:
```
⋊> which pyhindsight
which: no pyhindsight in ($HOME/bin:$HOME/.local/bin:$HOME/go/bin:/usr/local/bin:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

⋊> which hindsight
which: no hindsight in ($HOME/bin:$HOME/.local/bin:$HOME/go/bin:/usr/local/bin:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin)

⋊> ls -d $HOME/.local/lib/python3.11/site-packages/pyhindsight
$HOME/.local/lib/python3.11/site-packages/pyhindsight/

⋊> ls -l $HOME/.local/bin/hindsight.py
-rw-r--r--. 1 <user> <user> # <datetime> $HOME/.local/bin/hindsight.py
⋊> ls -l $HOME/.local/bin/hindsight_gui.py
-rw-r--r--. 1 <user> <user> # <datetime> $HOME/.local/bin/hindsight_gui.py

```

**Comment by obsidianforensics** [comment]
Sorry for the big delay here - I've done a new [release](https://github.com/obsidianforensics/hindsight/releases/tag/v2023.03) incorporating all the fixes (including #122) and pushed it to PyPI as well (https://pypi.org/project/pyhindsight/20230327.0/).

I hope this release fixes the packaging/install issues you all have identified - if not, please let me know. I appreciate the reports, it's hard to find these cases when it works on my systems.

As far as a leveldb viewer goes, I haven't found a free, usable, GUI version. https://github.com/cclgroupltd/ccl_chrome_indexeddb is what Hindsight uses for reading leveldb stores, which is what is being referenced above. I hadn't intended that (as part of Hindsight) to be called directly, so I'm not sure if it will work (the imports get a little weird). I'd suggest trying the scripts in their repo - that should be more stable.

**Comment by obsidianforensics** [comment]
Hindsight's code from [ccl_chrome_indexeddb](https://github.com/cclgroupltd/ccl_chrome_indexeddb) has been updated with their recent changes. As part of this, the `dump_*` scripts were removed from Hindsight, as they weren't intended to be called directly. If anyone needs to directly poke at leveldbs or indexeddb files, I'd highly recommend using the scripts included in [ccl_chrome_indexeddb](https://github.com/cclgroupltd/ccl_chrome_indexeddb).
